app/scripts/templates/settings/account_recovery/confirm_password.mustache

```diff
@@ -0,0 +1,30 @@
+<div id="account-recovery-confirm-password">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Re-enter password{{/t}}</h2>
+
+        <form novalidate>
+            <p class="description">
+                {{#t}}Enter your password to view your recovery key.{{/t}}
+            </p>
+            <p class="description">
+                <span class="note">{{#t}}Important:{{/t}}</span>
+                {{#t}}Access to your primary email and your recovery key will be needed to get back to your encrypted data.{{/t}}
+            </p>
+
+            <h2 class="email">{{email}}</h2>
+
+            <div class="input-row password-row">
+                <input type="password" class="password tooltip-below" id="password" placeholder="{{#t}}Enter Password{{/t}}" required pattern=".{8,}" autofocus/>
```

Comment:
The `autofocus` isn't kicking in here, not sure why. I have to manually click into the field.